cli/kata-env.go

```diff
@@ -319,6 +320,7 @@ func getHypervisorInfo(config oci.RuntimeConfig) HypervisorInfo {
 		Msize9p:           config.HypervisorConfig.Msize9p,
 		UseVSock:          config.HypervisorConfig.UseVSock,
 		MemorySlots:       config.HypervisorConfig.MemSlots,
+		EntropySource:     config.HypervisorConfig.EntropySource,
```

Comment:
- should this be displayed in `kata-env`?
- should we do any validation on this before handing to the hypervisor? (aka does it exist, is it a char device?)
  Looking at govmm, it also doesn't check this but that may well be by design since presumably qemu will :)
  https://github.com/intel/govmm/blob/master/qemu/qemu.go#L1035..L1042

/cc @markdryan

Reply:
@jodh-intel qemu complains about it,  `OCI runtime create failed: qemu-system-x86_64: -object rng-random,id=rng0,filename=/tmp/r: Could not open '/tmp/r': No such file or directory: unknown`